src/stdune/table.mli

```diff
@@ -0,0 +1,23 @@
+(**
```

Comment:
The first line should always be a one-liner synopsis. It is used by odoc when generating index pages.

Reply:
Changed.